data/ch4/ch4_load.sh

```diff
@@ -1,34 +1,59 @@
 #!/bin/bash
 
-# customizable variables
-DSBULK_PATH=/PATH/TO/dsbulk-1.3.4/bin
-DATADIR=/PATH/TO/graph-book/data/ch4
-
+# Please set the following variables
+DSBULK_PATH=/PATH/TO/dsbulk-1.3.4
 DEST_KS=neighborhoods_dev
 
+############################################
+# Automated data loading 
+# Assuming you set DSBULK_PATH and DEST_KS
+############################################
+
+# collect directory variables
+OLDWD="`pwd`"
+SCRIPTIR="`dirname $0`"
+cd "$SCRIPTDIR"
+DATADIR="`pwd`"
+
+# determine which dsbulk to use, in the case you already have it installed
+DSBULK="`which dsbulk`"
+if [ -z "$DSBULK" ]; then
+    if [ ! -f "$DSBULK_PATH/bin/dsbulk" ]; then
+        echo "Please set DSBULK_PATH variable to top-level path of DSBulk distribution"
+        echo "  or add directory with dsbulk to PATH, like, 'PATH=...dsbulk-1.3.4/bin:\$PATH'"
+        exit 1
+    fi
+fi
+DSBULK=$DSBULK_PATH/bin/dsbulk
+
+############################################
+# load the vertices
+############################################
 echo "Loading vertices into the graph $DEST_KS"
-cd $DSBULK_PATH
 
-./dsbulk load -k neighborhoods_dev -t Transaction -url /path/to/graph-book/data/ch4/Transactions.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t Vendor -url /path/to/graph-book/data/ch4/Vendors.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t Customer -url /path/to/graph-book/data/ch4/Customers.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t Loan -url /path/to/graph-book/data/ch4/Loans.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t Account -url /path/to/graph-book/data/ch4/Accounts.csv -header true -schema.allowMissingFields true
-./dsbulk load -k neighborhoods_dev -t CreditCard -url /path/to/graph-book/data/ch4/CreditCards.csv -header true -schema.allowMissingFields true
+$DSBULK load -k neighborhoods_dev -t Transaction -url /path/to/graph-book/data/ch4/Transactions.csv -header true -schema.allowMissingFields true
```

Comment:
We should remove `/path/to/graph-book/data/ch4/` from every line, as we're already in the folder with data.